lib/fathead/sass/trigger_words.txt

```diff
@@ -0,0 +1,2 @@
+sass
```

Comment:
@pjhampton FYI the "sass" trigger exists internally, before we had trigger_words.txt. I've added "scss" internally now 👍